features/bookmarks/bookmark_indexing.feature

```diff
@@ -165,3 +165,19 @@ Feature: Bookmark Indexing
     Then I should see "BookmarkedWork"
       And I should see "BookmarkedSeries"
       And I should see "BookmarkedExternalWork"
+
+  Scenario: When a bookmark with a tag is searched for, only bookmarks with
```

Comment:
This test would fit better into the `bookmark_filter.feature` file, this file is more about handling reindexing related to bookmark changes, not about how the the mapping is set up